### PR TITLE
split by feature=system

### PR DESCRIPTION
**Pull request by tharvik**
feature gate for `system`, to expose the config and avoid pulling all the `net` related deps.

### CODE REVIEW

**Comment by tharvik** [comment]
always the security issue

btw, I'm keeping an eye on it, there is a move upstream but it will probably be released in a while

File: Cargo.toml [hunk 1]
```diff
@@ -30,5 +31,5 @@ tokio = { version = "1.0", features = [ "macros" ] }
 [features]
 default = []
 cli = [ "structopt", "murmur/cli" ]
+system = ["drop/system", "murmur/system"]
```
Comment (ineiti) [inline Cargo.toml:34]: fomyruk: does that mean if you compile with the feature `system`, it also asks to compile `drop` and `murmur` with the `system` feature? Like `"crate/feature"`?

**Review by ineiti — APPROVED** [review]
LGTM - I suppose again that `src/system/mod.rs` is more or less the same as `src/lib.rs`.

File: Cargo.toml (hunk 1 above)
Comment (tharvik) [reply Cargo.toml:34]: Exactly!

**Pull request merged**